### PR TITLE
Create Contracts.md

### PR DESCRIPTION
**Pull request by mbusk88**
## Description

Start of documentation for contracts between PostOffice and sub domains.

Please be critical as other people outside Team Titans are going to read it.

## References

#230

### CODE REVIEW

File: docs/Contracts.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+| Recipient | string | optional | The Market Operator who sends the Dequeue message |
+
+<hr>
+
+<a name=".OtherUsers"></a>
+
+## Other users
+
+
+<a name=".DataAvailable.proto"></a>
```
Comment (xnetsrak) [inline docs/Contracts.md:96]: proto contract is: DataAvailableNotificationContract

File: docs/Contracts.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+All other users should skip the first section and only read section two where ProtoBuf contracts are described.
+
+## Table of Contents
+
+- [.NET users](#.NETusers)
+    - [DataAvailableNotificationDto](#DataAvailableNotificationDto)
+    - [DataBundleRequestDto](#DataBundleRequestDto)
+    - [DataBundleResponseDto](#DataBundleResponseDto)
+- [Other users](#.OtherUsers)
+    - [DataAvailable.proto](#DataAvailable.proto)
```
Comment (xnetsrak) [inline docs/Contracts.md:14]: DataAvailableNotificationContract

**Review by xnetsrak — CHANGES_REQUESTED** [review]

File: docs/Contracts.md (hunk 1 above)
Comment (mbusk88) [reply docs/Contracts.md]: Fixed

File: docs/Contracts.md (hunk 2 above)
Comment (mbusk88) [reply docs/Contracts.md]: Fixed

**Review by xnetsrak — APPROVED** [review]

**Review by defectiveAi — APPROVED** [review]

**Pull request merged**